test/unit/definitions/divorce/AuthorisationCaseField.test.js

```diff
@@ -26,7 +26,8 @@ describe('AuthorisationCaseField', () => {
         'CaseField',
         'CaseField-deemed-and-dispensed-nonprod',
         'CaseField-general-email-nonprod',
-        'CaseField-general-referral-nonprod'
+        'CaseField-general-referral-nonprod',
+        'CaseField-nonprod'
```

Comment:
why no flag?

Reply:
@yarpo `HelpWithFeesReferenceNumber` and  `FurtherPBAReferenceNumbers` are used in both deemed and dispensed and general referral features as they are reused. Seems the best place to me to put this.

Reply:
Updated